atlasdb-processors-tests/src/test/java/com/palantir/processors/AutoDelegateInterfaceTests.java

```diff
@@ -63,26 +65,35 @@ public void packagePrivateInterfacesGeneratePackagePrivateInterfaces() {
     @Test
     public void generatedInterfaceHasInterfaceMethods() {
         Set<String> generatedMethods = TestingUtils.extractMethods(AutoDelegate_TestInterface.class);
-        Set<String> originalMethods = TestingUtils.extractMethods(TestInterface.class);
+        Set<String> originalMethods = extractNonStaticMethods(TestInterface.class);
 
         assertThat(generatedMethods, hasItems(originalMethods.toArray(new String[0])));
     }
 
     @Test
     public void generatedInterfaceHasDelegateMethod() {
         Set<String> generatedMethods = TestingUtils.extractMethods(AutoDelegate_TestInterface.class);
-        Set<String> originalMethods = TestingUtils.extractMethods(TestInterface.class);
+        Set<String> originalMethods = extractNonStaticMethods(TestInterface.class);
 
         generatedMethods.removeAll(originalMethods);
         assertThat(generatedMethods.size(), is(1));
         assertThat(generatedMethods.iterator().next(), containsString("delegate"));
     }
 
+    @Test
+    public void generatedInterfaceDoesNotHaveStaticMethods() {
+        Set<String> generatedMethods = TestingUtils.extractMethods(AutoDelegate_TestInterface.class);
+        Set<String> originalStaticMethods = TestingUtils.extractMethodsSatisfyingPredicate(TestInterface.class,
+                method -> Modifier.isStatic(method.getModifiers()));
+
```

Comment:
`assertThat(Sets.intersection(generatedMethods, originalMethods)).isEmpty()`?

Reply:
Yep, that works. I'll stick with old-style here as having both junit and assertj styles in the same file gets annoying for imports.